### ISSUE TITLE
Image osm-entities.png missing

### ISSUE THREAD

**Issue by Nakaner** [osmcode/libosmium-manual#6]
osm-entities.png is missing in the out directory (and therefore not deployed on the server). It is referenced from [here](http://docs.osmcode.org/osmium-concepts-manual/#osm-entities) and should be at `http://docs.osmcode.org/osmium-concepts-manual/tmp/osm-entities.png`.

To fix this issue, the image has to copied to the `out` directory by the Makefile. The Makefile currently only copies it to the `tmp` directory (otherwise building the PDF would fail).

**Comment by joto** [comment]
Yeah. This whole building of the manuals is a horrible process. I am wondering whether it is worth it. Maybe we should just create HTML.

**Comment by Nakaner** [comment]
I tried to fix the bug in the Makefile yesterday but in the end I dropped my fix because it became too ugly.

I suggest to move all images to a separate directory called `img` and don't copy them to any other place during the build process. On server side the images (currently "the image") could be placed at a special `img` directory, too.

@joto wrote:
> Maybe we should just create HTML.

Some people are happy that pandoc creates TeX code in addition to the HTML. But I would not start crying if you drop the PDF output. :-)
